src/utils/helpers/createButton.jsx

```diff
@@ -0,0 +1,9 @@
+import "src/utils/helpers/createButtons.scss";
+
+export const createButton = (ICON, altText) => {
```

Comment:
component name should be PascalCase & props-name with camelCase is way better that this
```suggestion
export const CreateButton = (icon, altText) => {
```